pkg/cephmgr/osd/agent.go

```diff
@@ -357,6 +350,9 @@ func (a *osdAgent) configureDevices(context *clusterd.Context, devices map[strin
 	go func() {
 		defer connection.Shutdown()
 
+		// set the signal that the osd config is completed
+		defer close(a.osdsCompleted)
```

Comment:
what does this osdsCompleted mean?  we close it before we start the loop that does createOrStartOSD?